cmd/autoperibolossync/main.go

```diff
@@ -47,6 +48,7 @@ func parseOptions() options {
 
 	fs.StringVar(&o.peribolosConfig, "peribolos-config", "", "The peribolos configuration to be updated. Assuming that the file exists in the working directory.")
 	fs.StringVar(&o.releaseRepoPath, "release-repo-path", "", "Path to a openshift/release repository directory")
+	fs.StringVar(&o.whitelist, "whitelist", "", "Path to a whitelisted repositories file")
```

Comment:
Pls make this `--whitelist file` to be consistent everywhere.

Reply:
addressed